src/js/claims-status/containers/AppealsV2StatusPage.jsx

```diff
@@ -1,86 +1,48 @@
 import React from 'react';
 import PropTypes from 'prop-types';
-import { connect } from 'react-redux';
 
-import { getAppealsV2 } from '../actions/index.jsx';
 import { getStatusContents, getNextEvents } from '../utils/appeals-v2-helpers';
 
-import LoadingIndicator from '../../common/components/LoadingIndicator';
 import Timeline from '../components/appeals-v2/Timeline';
 import CurrentStatus from '../components/appeals-v2/CurrentStatus';
 import Alerts from '../components/appeals-v2/Alerts';
 import WhatsNext from '../components/appeals-v2/WhatsNext';
 import Docket from '../components/appeals-v2/Docket';
 
 /**
- * AppealsV2StatusPage is in charge of the layout of the status page and is the source of truth
- * for the redux state. All child components shouldn't need to be connected to the store.
+ * AppealsV2StatusPage is in charge of the layout of the status page
  */
-class AppealsV2StatusPage extends React.Component {
-  componentDidMount() {
-    // Make sure we grab the appeals if we don't have them already
-    // Useful if the user goes directly to the appeal status without going to the list first
-    if (this.props.appeal === AppealsV2StatusPage.defaultProps.appeal) {
-      this.props.getAppealsV2();
-    }
-  }
-
-  render() {
-    if (this.props.appealsLoading) {
-      return <LoadingIndicator message="Please wait while we load your appeal..."/>;
-    }
-    const { events, alerts, status } = this.props.appeal.attributes;
-    const { type, details } = status;
-    const currentStatus = getStatusContents(type, details);
-    const nextEvents = getNextEvents(type);
-    return (
-      <div>
-        <Timeline events={events}/>
-        <CurrentStatus
-          title={currentStatus.title}
-          description={currentStatus.description}/>
-        <Alerts alerts={alerts}/>
-        <WhatsNext nextEvents={nextEvents}/>
-        <Docket/>
-      </div>
-    );
-  }
-}
-
-AppealsV2StatusPage.defaultProps = {
```

Comment:
I remember adding this in here because something was breaking without any props...I'm guessing it was a weird time before we finished the action creator with the hardcoded appeals, but can you confirm that taking this out doesn't give you any console errors? (I'm still trying to get vets-api set up on this machine or I'd do it myself.)

Reply:
Yeah it doesn't. I definitely remember us going through this before - and we settled on needing a default empty state to compare against. I can't find any issues with doing things this way now, though I should probably spend a few minutes trying to figure out why it didn't used to work and why we should expect it to work now.

Reply:
I think I remember why we needed default props: We were grabbing `this.props.appeal.someValue` but couldn't because `appeal` (or whatever) wasn't passed in upon initial render (while we were still loading the appeals).

The way it's set up now negates the need to have these default empty props because this component won't even render until we actually have an appeal.